packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -211,6 +213,38 @@ sealed class _DebugSemanticsRoleChecks {
     node.visitChildren(validateRadioGroupChildren);
     return error;
   }
+
+  static FlutterError? _semanticsMenu(SemanticsNode node) {
+    if (node.childrenCount < 1) {
+      return FlutterError('a menu cannot be empty');
+    }
+
+    return null;
+  }
+
+  static FlutterError? _semanticsMenuBar(SemanticsNode node) {
+    if (node.childrenCount < 1) {
+      return FlutterError('a menu bar cannot be empty');
+    }
+
+    return null;
+  }
+
+  static FlutterError? _semanticsMenuItemCheckbox(SemanticsNode node) {
+    if (!node.hasFlag(SemanticsFlag.hasCheckedState)) {
```

Comment:
we should use node.getSemanticsData to access the property, here and every where